test/commands/init.test.ts

```diff
@@ -78,7 +78,7 @@ services:
       expect(component_config.services['elasticsearch'].environment['ES_JAVA_OPTS']).eq('-Xmx256m -Xms256m');
       expect(component_config.services['elasticsearch'].environment['ELASTIC_PASSWORD']).eq('changeme');
       expect(component_config.services['elasticsearch'].environment['DISCOVERY_TYPE']).eq('single-node');
-      expect(component_config.services['elasticsearch'].environment['TEST_NUMBER']).eq('3000');
+      expect(component_config.services['elasticsearch'].environment['TEST_NUMBER']).eq(3000);
```

Comment:
This doesn't look right - environment variables are always strings. Pretty sure k8s won't accept numbers or booleans. Undo these changes - the only one you need to make was in `getExtraEnvironmentVariables`. 

Also can you remove the unused `getExtraEnvironmentVariables` method in `deploy.ts`?